conf/application.conf

```diff
@@ -164,6 +164,19 @@ microservice {
       host = "localhost"
       port = 9490
     }
+
+    transit-movements {
```

Comment:
Note: we need to update this config in `app-config-base` for the in-environment values.